README.md

```diff
@@ -1,50 +1,166 @@
+```
+
+2. Go to the project directory
+
+```bash
+  cd shorti-url-shortener
+```
+
+3. Install dependencies in both client and server
+
+```bash
+  yarn install
+```
+
+4. Start the server
+
+```bash
+  yarn start  ( for both client and server)
+  yarn client ( client only)
+  yarn server (server only)
+```
+ **Note:** Create and fill in server/.env and client/.env 
+
+ More details [here](https://github.com/webdev-20/shorti-url-shortener/blob/main/CONTRIBUTE.md)
+
+ <p align="right">(<a href="#top">back to top</a>)</p>
+ 
+## Contributing
+
+We welcome your contribution!
+
+Please get the issue assigned to yourself before working on it, to avoid duplicate efforts.
+
+Any Suggestions, please fork the repo and create a pull request.
+Or create an issue with tag - **enhancement**
+
+1. Fork the repository
+2. Clone it and navigate to your local repository
+3. Create your feature branch, example:- fix/header
+4. Stage changes & commit
+5. Create a pull request
+
+See [contributing.md](https://github.com/webdev-20/shorti-url-shortener/blob/main/CONTRIBUTE.md) for ways to get started and more details.
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## API Documentation
+
+Details of API endpoints can be viewed here
+[Documentation](https://localhost:4002/api/docs)
+
+## Features
+
+- [ ]  (database/backend) link user and link objects
+- [ ]  frontend test(cypress)
+- [ ]  backend tests (jest)
+
+Visit the [open issues](https://github.com/webdev-20/shorti-url-shortener/issues) for a full list of proposed features. 
+
+## Live Site
+
+Visit our production site here - 
 https://shorti.onrender.com/
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## License
+
+[MIT](https://choosealicense.com/licenses/mit/)
+
+## Contacts
+
+Cherylli - [@twitter_handle](https://twitter.com/twitter_handle) - email@email_client.com
+
+Klezi10 - [@twitter_handle](https://twitter.com/twitter_handle) - email@email_client.com
+
+Project Link: [https://github.com/webdev-20/shorti-url-shortener](https://github.com/webdev-20/shorti-url-shortener)
+
+## Acknowledgements
 
-## Design
-https://www.figma.com/file/SP0iLbONWkjZ0cTlpBwSPR/Shorti-Copy?node-id=0%3A1
-
-## API Docs: 
-http://localhost:4002/api/docs/
-
-# Contributors
-
-Thanks for participating in The Lassie Lounge Hacktoberfest 2022 project
-
-<table>
-  <tr>
-    <td>
-     <img src="https://github.com/klezi10.png" width="50"/>
-      <br/>
-        <a href="https://github.com/klezi10">klezi10</a>
-     </td>
-     <td>
-      <img src="https://github.com/cherylli.png" width="50"/>
-      <br/>
-        <a href="https://github.com/cherylli">cherylli</a>
-     </td>
-     <td>
-      <img src="https://img.icons8.com/color/344/circled-user-female-skin-type-4--v1.png" width="50"/>
-      <br/>
-        <a href="#">Person 3</a>
-     </td>
-  </tr>
-</table>
+ - [Design](https://www.figma.com/file/SP0iLbONWkjZ0cTlpBwSPR/Shorti-Copy?node-id=0%3A1)
```

Comment:
The design probably shouldn't need be in the readme as it is only for development